### PR TITLE
fix deprecations

### PR DESCRIPTION
**Pull request by abhijithch**
Fixes deprecations on `Julia v0.5`.

### CODE REVIEW

**Comment by mlubin** [comment]
Thanks but this breaks compatibility with 0.4, see the travis failure.

File: src/MathProgBase.jl [hunk 1]
```diff
@@ -4,6 +4,7 @@ module MathProgBase
 
 include(joinpath(dirname(@__FILE__),"SolverInterface","SolverInterface.jl"))
 using .SolverInterface
+using Compat
```
Comment (tkelman) [inline src/MathProgBase.jl]: https://github.com/JuliaOpt/MathProgBase.jl/blob/master/REQUIRE

Comment (abhijithch) [inline src/MathProgBase.jl]: Among all the tags in `Compat` there is no commit which includes `issymmteric`, @tkelman how do I find the version here?

**Comment by abhijithch** [comment]
@mlubin I have gotten the fix for `v0.4`, just figuring out the right `Compat` version, sorry for the last commit which actually didnt do much.

File: src/MathProgBase.jl (hunk 1 above)
Comment (tkelman) [inline src/MathProgBase.jl]: > Among all the tags in `Compat` there is no commit which includes `issymmteric`

Why do you think that?

See https://github.com/JuliaLang/Compat.jl/blame/57122c4d8275f7ba41229c74f01db490311787aa/README.md#L192, so it was introduced in https://github.com/JuliaLang/Compat.jl/commit/3e53ba57e150229bc3590398b5040f2ff7c28fbc which you can see was first available in 0.7.13.

Comment (abhijithch) [inline src/MathProgBase.jl]: I just looked in the tags. Thanks.

**Comment by abhijithch** [comment]
I have done the necessary updates. Could this be merged?

**Comment by mlubin** [comment]
Thanks @abhijithch

**Pull request merged**